velox/exec/tests/RowNumberTest.cpp

```diff
@@ -218,4 +275,40 @@ TEST_F(RowNumberTest, largeInput) {
   testLimit(5'000);
 }
 
+TEST_F(RowNumberTest, reclaimFail) {
+  auto spillDirectory = exec::test::TempDirectoryPath::create();
+  auto data = makeRowVector({
+      makeFlatVector<int64_t>({1, 2, 1, 2, 1, 2, 1}),
+      makeFlatVector<int64_t>({1, 2, 3, 4, 5, 6, 7}),
+  });
+
+  createDuckDbTable({data});
```

Comment:
This statement is not needed since this test is not expected to produce results that need comparing.